src/ec2/utils.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Ec2Instance } from '../shared/clients/ec2Client'
+
+export function getIconForInstanceStatus(instance: Ec2Instance) {
+    return `$(${getIconCodeForInstanceStatus(instance)})`
+}
+
+export function getIconCodeForInstanceStatus(instance: Ec2Instance) {
```

Comment:
Premature namespacing. Unlikely to be other icons for EC2 nodes, so we don't need to over-explain in the function name.

```suggestion
export function getIconCode(instance: Ec2Instance) {
```